nautobot/extras/migrations/0022_objectchange_object_datav2.py

```diff
@@ -0,0 +1,21 @@
+# Generated by Django 3.1.14 on 2022-02-18 17:25
+
+import django.core.serializers.json
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ("extras", "0021_customfield_changelog_data"),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name="objectchange",
+            name="object_datav2",
+            field=models.JSONField(
+                blank=True, editable=False, encoder=django.core.serializers.json.DjangoJSONEncoder, null=True
```

Comment:
I think this needs to be:

```suggestion
                blank=True, editable=False, encoder=nautobot.core.celery.NautobotKombuJSONEncoder, null=True
```